r/runtime/build_info.json

```diff
@@ -0,0 +1,12 @@
+{
+    "package_name" : "runtime",
+    "github_url":"https://github.com/go-openapi/runtime",
```

Comment:
Please provide the sample version of the package. (In your case, it is v0.19.0).
Please follow [build_info_template.json](https://github.com/ppc64le/build-scripts/blob/master/templates/build_info_template.json), for further reference.